src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -871,7 +871,11 @@ export class MatrixRTCSession extends TypedEventEmitter<MatrixRTCSessionEvent, M
             logger.info(`Memberships for call in room ${this.room.roomId} have changed: emitting`);
             this.emit(MatrixRTCSessionEvent.MembershipsChanged, oldMemberships, this.memberships);
 
-            if (this.isJoined() && !this.memberships.some(this.isMyMembership)) {
+            if (
+                this.isJoined() &&
+                oldMemberships.some(this.isMyMembership) &&
```

Comment:
Looks good but I'd suggest a comment here, as reading this I'd probably assume this check is redundant

Reply:
d48ee9220